### PR TITLE
Yolov8 quantize with accuracy control and run with yolov8n-seg model

### PR DESCRIPTION
**Pull request by yangsu2022**
### Changes
This new PR is based on the feedback of the closed one.
Add nncf.quantize_with_accuracy_control instead of nncf.quantize
Do segmentation with coco-128-seg dataset

### Reason for changes
Extend the usage of POT with accuracy control

### CODE REVIEW

**Comment by MaximProshin** [comment]
Initial PR https://github.com/openvinotoolkit/nncf/pull/1580

File: examples/post_training_quantization/openvino/yolov8_quantize_with_accuracy_control/main.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+    for batch_i, batch in enumerate(data_loader):
+        if num_samples is not None and batch_i == num_samples:
+            break
+        batch = validator.preprocess(batch)
+        results = compiled_model(batch["img"])
+        if num_outputs == 1:
+            preds = torch.from_numpy(results[compiled_model.output(0)])
+        else:
+            preds = [torch.from_numpy(results[compiled_model.output(0)]), [
+                torch.from_numpy(results[compiled_model.output(1)])]]
```
Comment (alexsu52) [inline examples/post_training_quantization/openvino/yolov8_quantize_with_accuracy_control/main.py:51]:
```suggestion
            preds = [
                torch.from_numpy(results[compiled_model.output(0)]),
                [torch.from_numpy(results[compiled_model.output(1)])]
            ]
```

**Comment by yangsu2022** [comment]
@alexsu52 Thanks for the nice suggestion. Please check my new commits.

**Review by alexsu52 — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**